gravitee-am-model/src/main/java/io/gravitee/am/model/MFASettings.java

```diff
@@ -39,8 +39,11 @@ public class MFASettings {
     private String skipStepUpAuthentication;
```

Comment:
we can remove it or move it to new enrollment settings

Reply:
removed for now